[IMP] marketing automation: update understanding_metrics for 16 (#5161)
odoo/documentation: content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

(represented by a :guilabel:`pie chart` icon), displaying related metrics as a simple line graph.
The success metrics are represented in `green` and the rejected metrics are represented in `red`.

A numerical representation of both (:guilabel:`Success` and :guilabel:`Rejected`) is presented to
the right of the line graph, as well.

.. tip::
   Hovering over any point in the line graph of the activity block reveals a notated breakdown of
   data for that specific date.

Beneath the graph in the activity block, *if* the activity type is *Email* or *SMS*, there is a line
of accessible data figures that are directly related to the campaign activity. These detailed
statistics provide a bird's eye view of the following metrics: :guilabel:`Sent` (numerical),
:guilabel:`Clicked` (percentage), :guilabel:`Replied` (percentage), and :guilabel:`Bounced`
(percentage).

.. tip::
   Clicking any of those stats on the :guilabel:`DETAILS` line, beneath the line graph, reveals a
   separate page containing every specific record for that particular data point.

Activity filter tab
-------------------

Next to the :guilabel:`graph` tab on the activity block, there's the option to open a

> :guilabel:`filter` tab (represented by a :guilabel:`filter/funnel` icon).

Reviewer: ```suggestion
:guilabel:`Filter` tab (represented by a :guilabel:`filter/funnel` icon).
```

.. image:: understanding_metrics/activity-filter-tab.png
   :align: center
   :alt: What a campaign activity filter tab looks like in Odoo Marketing Automation.

Clicking the :guilabel:`filter` tab on an activity block, reveals what the specific filters are for
that particular campaign activity, and how many records in the database match that specific

> criteria.

Reviewer: What are some examples of filters people would want to apply to get the marketing campaign info they need?

This information isn't necessary for you to add here, but was just something I was personally curious about (:

.. tip::

>    Clicking on the :guilabel:`records` link beneath the displayed filter reveals a separate pop-up

Reviewer: ```suggestion
   Clicking the :guilabel:`records` link beneath the displayed filter reveals a separate pop-up
```

   window containing a list of all the records that match that specific campaign activity rule(s).

Link tracker
============

Odoo tracks all URLs used in marketing campaigns. To access and analyze those URLs, navigate to
:menuselection:`Marketing Automation app --> Reporting --> Link Tracker`. Doing so reveals a
:guilabel:`Link Statistics` page, wherein all campaign-related URLs can be analyzed.

.. image:: understanding_metrics/campaign-link-tracker.png
   :align: center
   :alt: What a campaign activity filter tab looks like in Odoo Marketing Automation.

The default view on the :guilabel:`Link Statistics` page is the :guilabel:`Bar Chart` view, but
there are different view options available in the upper-left corner. There is the option to view the

> statistics as a :guilabel:`Line Chart` or :guilabel:`Pie Chart`.

Reviewer: Although it's useful to know how the data is displayed, it's also useful to know *what* data is being displayed.
For instance, why is it important for marketers to know which links their prospective customers clicked on to see the marketing campaign?

Author: I totally understand this suggestion - completely get where you're coming from. However, in the past, I've been instructed to stick to "how" to access data -- and avoid explaining "why" certain data is useful -- because the user will determine their own value to the data presented to them, if that makes sense. So, with that in mind, I think I'll keep this as-is, but if it comes up in another review, I'll add it in. Thanks! 👍

can be put into :guilabel:`Descending` or :guilabel:`Ascending` order.

To the far-left of the view options, there is the :guilabel:`Measures` drop-down menu. When clicked,
the options to view the :guilabel:`Number of Clicks` or total :guilabel:`Count` are available. And,
to the right of the :guilabel:`Measures` drop-down menu, there's the ability to add any data to a
spreadsheet by clicking the :guilabel:`Insert in Spreadsheet` button.

Also, in the upper-right corner of the :guilabel:`Link Statistics` page, to the far-right of the
search bar, there are additional view options to choose from: the default :guilabel:`Graph` view,
the :guilabel:`Pivot` table view, and the :guilabel:`List` view.

Traces
======

Odoo tracks all activities used in every marketing campaign. The data related to these activities
can be accessed and analyzed in the :guilabel:`Traces` page, which can be found by navigating to
:menuselection:`Marketing Automation app --> Reporting --> Traces`.

.. image:: understanding_metrics/traces-page-marketing-automation.png
   :align: center
   :alt: The Traces page in the Odoo Marketing Automation application.

> The default view on the :guilabel:`Traces` page is the :guilabel:`Bar Chart` view, but there are
> different view options available in the upper-left corner. There is the option to view the
> statistics as a :guilabel:`Line Chart` or :guilabel:`Pie Chart`.

Reviewer: Although this paragraph isn't wrong, I felt like I was reading a description of the UI instead of about the function of the feature. I tried to describe what I think I'm seeing on this Link Statistics page in my own words, but I'm not really sure... 

Feel free to correct anything!

```suggestion
On the :guilabel:`Link Statistics` page, click-through engagement with each activity in the
marketing campaign is displayed in the :guilabel:`Bar Chart` view, by default. Alternative options
to view this data in a :guilabel:`Line Chart` or :guilabel:`Pie Chart` is available in the
upper-left corner.
```
